src/swap/v1_1/index.ts

```diff
@@ -62,32 +68,25 @@ async function signTxns({
 
 async function generateTxns({
   client,
-  pool,
+  quote,
```

Comment:
We can alias the variable like this so that it doesn't look ambiguous below
```suggestion
  quote: quoteAndPool,
```